.drone.yml

```diff
@@ -1,7 +1,21 @@
 build:
-  image: ensime/ensime:v1.x-cache
-  pull: true
-  commands:
-    - pip install virtualenv python-coveralls
-    - make test
+  quality:
+    image: ensime/ensime:v1.x-cache
+    pull: true
+    commands:
+      - pip install --upgrade virtualenv
+      - make lint
+      - make format ;
+        if $(! git diff --exit-code --quiet) ; then
+          echo "Code formatting does not meet the project's standards:" ;
```

Comment:
This might be clarified somewhere in the Contributing Guidelines to avoid frustrating rejections.
Could it be enforced in a Git hook maybe? Sounds like a job for machines :)